plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraMetadata.java

```diff
@@ -351,11 +377,11 @@ public ConnectorInsertTableHandle beginInsert(ConnectorSession session, Connecto
         }
 
         CassandraTableHandle table = (CassandraTableHandle) tableHandle;
```

Comment:
If we can change the above line as `CassandraNameRelationHandle table = ((CassandraTableHandle) tableHandle).getRequiredNamedRelation()

Then most of git diff can be avoided right ?